src/manager.cpp

```diff
@@ -112,15 +126,87 @@ void Manager::SetTimerToDetectSVPDOnDbus()
 }
 #endif
 
-void Manager::updateKeyword(const types::Path i_path,
-                            const types::VpdData i_data, const uint8_t i_target)
+int Manager::updateKeyword(types::Path i_path, const types::VpdData i_data)
```

Comment:
updateKeyword API actual implementation should be added in 'utils', so that BackupAndRestore implementation can make utils updateKeyword call.
I am seeing this implementation.